### PR TITLE
Create mysterious-halfling.md

### PR DESCRIPTION
**Pull request by malpinder**
Paging @fatbusinessman

### CODE REVIEW

File: pals/mysterious-halfling.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#### Feats
+* Lost in the Crowd
+
+#### Equipment
+* Leather Armor
+* 5x Dagger
+* 10x Shuriken
+* 3x Torches
+* Adventurer's Kit
+* Thieves' tools
```
Comment (AgentAntelope) [inline pals/mysterious-halfling.md:64]: One adventurer, multiple thieves?

File: pals/mysterious-halfling.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+
+# ??? It Is A Mystery
```
Comment (malpinder) [inline pals/mysterious-halfling.md]: You should come up with a name; I've been using heptambic nearly-alliterative for halfing NPCs (eg Grumblefront Goldenpalm, Riddley Rumblefruit) if you'd like to fit in, but it's fine to be different (halflings come from lots of different countries & cultures, after all)

File: pals/mysterious-halfling.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+> ??? This Is Also A Mystery
+
+#### Species, Class, Level:
+Halfing Rogue, Level 1
+
+#### Languages known:
+Common, +1 other
```
Comment (malpinder) [inline pals/mysterious-halfling.md]: You can pick from:
* Dwarven (what dwarves speak)
* Elven (what elves & fey creatures speak)
* Draconic (dragons, kobolds, reptile people etc)
* Giant (giants, orcs, hobgoblins)
* Goblin (goblins, bugbears, and almost every orc & hobgoblin will also know this)
* Deep Speech (mostly-evil things from the Underdark)
* Primordial (elementals and things from Beyond Time)

Whatever you pick will probably hint at a backstory, if/when we get that far.

File: pals/mysterious-halfling.md (hunk 1 above)
Comment (malpinder) [reply pals/mysterious-halfling.md:64]: Yes, it's like matryoshka dolls, you release a smaller thief into the lock and they have their own heist adventure to break it.

**Pull request merged**